client/components/profile/EditProfileForm.jsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import RaisedButton from 'material-ui/RaisedButton';
+import PersonAdd from 'material-ui/svg-icons/social/person-add';
+import Snackbar from 'material-ui/Snackbar';
+import lodash from 'lodash';
+import {
+  withRouter,
+} from 'react-router-dom';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import LockIcon from 'material-ui/svg-icons/action/lock';
+import RoleIcon from 'material-ui/svg-icons/action/group-work';
+import AccessibilityIcon from 'material-ui/svg-icons/action/accessibility';
+import CircularProgress from 'material-ui/CircularProgress';
+import FormElements from '../common/FormTextFields';
+import { updateUserAction, getUserAction } from '../../actions/users.action';
+
+const styles = {
+  formStyle: {
+    margin: 15,
+    color: 'white',
+  },
+  button: {
+    margin: 1,
+    width: '',
+    display: 'inline-block',
+    position: 'relative',
+  },
+  formElement: {
+    margin: 5,
+  },
+};
+
+
+class EditProfileForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      errors: '',
+      isLoading: false,
+      redirect: false,
+      snackBarOpen: false,
+      firstname: '',
```

Comment:
Can you put all of the user state in to the user object
```
user: {}
```
and access it with `this.state.user[<property>]`